src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -154,3 +164,21 @@ def load_deaths(self, location_id: int) -> pd.DataFrame:
     def load_reff(self, location_id: int) -> pd.DataFrame:
         file = self.forecast_paths.get_output_reff(location_id=location_id)
         return pd.read_csv(file)
+
```

Comment:
Per @collijk I was to PR to this branch even though forecasting is not yet fully integrated.

It was very explicit that moving forward forecasting will _also_ be by `draw_id` (and `scenario`) and that a post-processing step would re-orient the data into by location.

I took a best guess at the methods and their arguments and implemented them with an easy to remove `_futurerefactor` suffix.